engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/DevProcessConfigCreator.scala

```diff
@@ -491,9 +492,20 @@ class SimpleTypesCustomStreamTransformer extends CustomStreamTransformer with Se
 // In services all parameters are lazy evaluated
 class SimpleTypesService extends Service with Serializable {
   @MethodToInvoke
-  def invoke(@ParamName("booleanParam") booleanParam: Boolean,
-             @ParamName("stringParam") string: String,
-             @ParamName("intParam") intParam: Int,
+  def invoke(@ParamName("booleanParam")
```

Comment:
Can you also add parameter with fixed values? So in FE change won't be needed to change this definition.

Reply:
Done